src/main/java/builder/FourthMain.java

```diff
@@ -0,0 +1,18 @@
+package builder;
+
+public class FourthMain {
+    public static void main(String[] args) {
+    
+    //Build a SQL Connection
```

Comment:
the code should be self explanatory. Only add comments if there is no other option